ui/config-overrides.js

```diff
@@ -183,10 +204,10 @@ module.exports = override(
     )
   ),
   disableMinimizeByEnv(),
-  addExtraEntries(),
   supportDynamicPublicPathPrefix(),
   overrideProcessEnv({
     REACT_APP_RELEASE_VERSION: JSON.stringify(getInternalVersion()),
   }),
-  injectDistroToHTML
+  injectDistroToHTML,
+  addExtraEntries()
```

Comment:
make sure `addExtraEntries()` run before `injectDistroToHtml` (the running order is reverse), else `injectDistroToHtml` only works for index.html, but not diagnoseReport.html

Reply:
How about adding this comment in the code.